lib/Doctrine/ruleset.xml

```diff
@@ -100,12 +100,16 @@
                 name="forbiddenAnnotations"
                 type="array"
                 value="
+                    @api,
                     @author,
+                    @category,
                     @copyright,
                     @created,
                     @license,
                     @package,
                     @since,
+                    @subpackage,
+                    @uses,
```

Comment:
This shouldn't be added IMO since PHPUnit uses it for explicit coverage definition (`@covers` and `@uses`).

Reply:
Oh, I see it https://phpunit.de/manual/current/en/appendixes.annotations.html, sorry!

Reply:
👍 `@uses` is already whitelisted btw.